Dockerfile

```diff
@@ -1,27 +1,22 @@
-ARG IMAGE=intersystemsdc/irishealth-community
-ARG IMAGE=intersystemsdc/iris-community
 ARG IMAGE=intersystemsdc/iris-community:preview
 FROM $IMAGE
 
 WORKDIR /home/irisowner/dev
 
-ARG TESTS=0
-ARG MODULE="iris-python-template"
-ARG NAMESPACE="USER"
-
 # create Python env
 ## Embedded Python environment
 ENV IRISUSERNAME "SuperUser"
 ENV IRISPASSWORD "SYS"
-ENV IRISNAMESPACE "USER"
+ENV IRISNAMESPACE "IRISAPP"
 ENV PYTHON_PATH=/usr/irissys/bin/
-ENV PATH "/usr/irissys/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/home/irisowner/bin"
+ENV LIBRARY_PATH=${ISC_PACKAGE_INSTALLDIR}/bin:${LIBRARY_PATH}
```

Comment:
What is it for?

Reply:
it's to make sure that the iris embedded python module (import iris) works with python3 instead of irispython